app/controllers/pxe_controller.rb

```diff
@@ -156,18 +157,20 @@ def replace_right_cell(options = {})
                         end
     when :customization_templates_tree
       presenter.update(:main_div, r[:partial => "template_list"])
-      nodes = nodetype.split('_')
-      if @in_a_form
-        right_cell_text =
-          if @ct.id.blank?
-            _("Adding a new %{model}") % {:model => ui_lookup(:model => "PxeCustomizationTemplate")}
-          else
-            @edit ? _("Editing %{model} \"%{name}\"") % {:name  => @ct.name, :model => ui_lookup(:model => "PxeCustomizationTemplate")} :
-                    _("%{model} \"%{name}\"") % {:name  => @ct.name, :model => ui_lookup(:model => "PxeCustomizationTemplate")}
-          end
-        # resetting ManageIQ.oneTransition.oneTrans when tab loads
-        presenter.reset_one_trans
-        presenter.one_trans_ie if %w(save reset).include?(params[:button]) && is_browser_ie?
+      presenter.update(:paging_div, r[:partial => "layouts/x_pagingcontrols"])
+      unless params[:ppsetting]
```

Comment:
yea right, it's not needed, I've put it there previously to avoid nil on local variable ```nodes```, that I found as unused later and removed it in this PR .. so the conditional is not needed here .. good catch ;-)